haystack/management/commands/update_index.py

```diff
@@ -222,7 +227,7 @@ def update_backend(self, label, using):
             total = qs.count()
 
             if self.verbosity >= 1:
-                print(u"Indexing %d %s" % (total, force_text(model._meta.verbose_name_plural)))
+                logger.info(u"Indexing {} {}".format(total, force_text(model._meta.verbose_name_plural)))
```

Comment:
The logger includes the timestamps to the messages.  But maybe i'll add the overall process time to the bottom.